modules/fedimint-ln/src/config.rs

```diff
@@ -18,16 +19,21 @@ pub struct LightningConfig {
     pub consensus: LightningConfigConsensus,
 }
 
-#[derive(Debug, Clone, Serialize, Deserialize)]
+#[derive(Debug, Clone, Serialize, Deserialize, Encodable)]
 pub struct LightningConfigConsensus {
     /// The threshold public keys for encrypting the LN preimage
     pub threshold_pub_keys: threshold_crypto::PublicKeySet,
-    /// The number of decryption shares required
-    pub threshold: usize,
     /// Fees charged for LN transactions
     pub fee_consensus: FeeConsensus,
 }
 
+impl LightningConfigConsensus {
+    /// The number of decryption shares required
+    pub fn threshold(&self) -> usize {
```

Comment:
That `+1` always gets me. It starts with https://docs.rs/hbbft/latest/hbbft/ , where they already use `+ 1` all over the place. They should have made the `threshold` inclusive, because that would be more intuitive.

The fact that we have some "threshold"s mean "other thresholds + 1" is kind of crazy.

We might want to name them `threshold_inclusive" or something. Otherwise it doesn't make sense.

Reply:
It's just as crazy in the academic literature. I once had to change the definition of `t` in a paper because my prof liked one version over the other which I had found in another paper :man_facepalming: